code/src/components/QuestLog.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+
+import { Quest } from 'components/Quest'
+
+export const QuestLog = () => {
+  const questsList = useSelector((state) => state.quests.questsList)
+
+  const mainQuests = []
+  const sideQuests = []
+  const miscellaneousQuests = []
+
+  questsList.forEach((e, index) => {
+    if (e.category === 'Main') {
+      mainQuests.push(index)
+    } else if (e.category === 'Side') {
+      sideQuests.push(index)
+    } else {
+      miscellaneousQuests.push(index)
+    }
+  })
```

Comment:
I think it would've been nicer to use `.filter` on your `questsLists` to find the items you're looking for:

```
const mainQuests = questsList.filter((quest) => quest.category === 'Main'))
const sideQuests = questsList.filter((quest) => quest.category === 'Side'))
const miscellaneousQuests = questsList.filter((quest) => !['Main', 'Side'].includes(quest.category)))
```

... Or just use 3 selectors and get what you want directly from redux.